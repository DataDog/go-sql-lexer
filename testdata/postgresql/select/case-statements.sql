SELECT name, CASE WHEN age < 18 THEN 'minor' ELSE 'adult' END FROM users;
