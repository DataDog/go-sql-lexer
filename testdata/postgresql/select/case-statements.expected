[
    {
        "expected": "SELECT name, CASE WHEN age < ? THEN ? ELSE ? END FROM users",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]